tensorflow/lite/micro/micro_profiler.cc

```diff
@@ -67,4 +68,54 @@ void MicroProfiler::LogCsv() const {
 #endif
 }
 
+void MicroProfiler::LogTicksPerTagCsv() {
+#if !defined(TF_LITE_STRIP_ERROR_STRINGS)
+  MicroPrintf(
+      "\"Unique Tag\",\"Total ticks across all events with that tag.\"");
+  int total_ticks = 0;
+  for (int i = 0; i < num_events_; ++i) {
+    uint32_t ticks = end_ticks_[i] - start_ticks_[i];
+    if (tags_[i] == nullptr) {
+      MicroPrintf("Event does not have a tag name.");
+      continue;
+    }
```

Comment:
can this happen? seems like this should be a DCHECK and not an error that we want to print and continue on.

Reply:
Added a DCHECK. Thank you for the suggestion.